internal_macros/src/lib.rs

```diff
@@ -616,3 +616,42 @@ pub fn llvm_enum(attribute_args: TokenStream, attributee: TokenStream) -> TokenS
     };
     q.into()
 }
+
+/// This macro adds shared code to get/set section of global values
+#[proc_macro_derive(Section)]
+pub fn section_derive(input: TokenStream) -> TokenStream {
```

Comment:
This is a neat idea but I'm a little concerned that sharing code through derive macros will make the code harder to maintain (for example, if there's a bug here, or if an LLVM fn is deprecated for a newer version, we have to release a new inkwell_internals ver before inkwell)

Our current paradigm is to add methods to the underlying `Value` type, and expose them in the wrapping `*Value` types - I think we should stick to that here

Reply:
For example: https://github.com/TheDan64/inkwell/blob/master/src/values/float_value.rs#L48

Reply:
Done.